src/AltinnCore/Runtime/Controllers/PartiesController.cs

```diff
@@ -2,55 +2,133 @@
+using AltinnCore.RepositoryClient.Model;
+using AltinnCore.ServiceLibrary.Enums;
 using AltinnCore.ServiceLibrary.Models;
 using AltinnCore.ServiceLibrary.Services.Interfaces;
+using Common.Helpers;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using ServiceLibrary.Models;
 
 namespace AltinnCore.Runtime.Controllers
 {
     /// <summary>
     /// Handles party related operations
     /// </summary>
     [Authorize]
-    public class PartiesController : Controller
+    [ApiController]
+    public class PartiesController : ControllerBase
     {
         private readonly ILogger _logger;
         private readonly IAuthorization _authorization;
         private readonly UserHelper _userHelper;
+        private readonly IRepository _repository;
+        private readonly IProfile _profile;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="PartiesController"/> class
         /// <param name="logger">The logger</param>
         /// <param name="authorization">the authorization service handler</param>
         /// <param name="profileService">the profile service</param>
         /// <param name="registerService">the register service</param>
-        /// <param name="userHelper">the user helper</param>
+        /// <param name="repository">The repository service</param>
         /// </summary>
         public PartiesController(
                     ILogger<PartiesController> logger,
                     IAuthorization authorization,
                     IProfile profileService,
-                    IRegister registerService)
+                    IRegister registerService,
+                    IRepository repository)
         {
             _logger = logger;
             _authorization = authorization;
             _userHelper = new UserHelper(profileService, registerService);
+            _repository = repository;
+            _profile = profileService;
         }
 
         /// <summary>
-        /// Gets the lsit of parties the user can represent
+        /// Gets the list of parties the user can represent
         /// </summary>
         /// <returns></returns>
-        [HttpGet]
+        [HttpGet("/Get")]
         public IActionResult Get()
         {
             UserContext userContext = _userHelper.GetUserContext(HttpContext).Result;
             List<Party> partyList = _authorization.GetPartyList(userContext.UserId);
             return Ok(partyList);
         }
+
+        /// <summary>
+        /// Validates party and profile settings before the end user is allowed to instantiate a new app instance
+        /// </summary>
+        /// <param name="org">The organization<param>
+        /// <param name="app">The application<param>
+        /// <param name="partyId">The selected partyId<param>
+        /// <returns>A validation status</returns>
+        [HttpPost("{org}/{app}/api/v1/parties/validateInstantiation")]
```

Comment:
Maybe move the routing to Startup, so all routing is co-located.

Reply:
I dont know why we insist on these routes in startup, I hate to have to look at two different places to find my route.

Reply:
I think the most important thing is to be consistent, and placing the routing in the class for only one of the controllers in runtime causes confusion when having to maintain the code later.

Reply:
Attribute-based routing is the recommended way to route your api-controllers. To make a conventional route in the startup.cs file. Do you have any input on this @TheTechArch ?

Reply:
I feel like attribute based routing should be used, at least for our restful api's.

Reply:
We should use attribute routing. It is the only thing swagger support. The new API we will create for instance will use attribute.